contribs/github-bot/config.go

```diff
@@ -0,0 +1,100 @@
+package main
+
+import (
+	"github.com/gnolang/gno/contribs/github-bot/client"
+	c "github.com/gnolang/gno/contribs/github-bot/conditions"
+	r "github.com/gnolang/gno/contribs/github-bot/requirements"
+)
+
+// Automatic check that will be performed by the bot
+type automaticCheck struct {
+	Description string
+	If          c.Condition   // If the condition is met, the rule is displayed and the requirement is executed
+	Then        r.Requirement // If the requirement is satisfied, the check passes
+}
+
+// Manual check that will be performed by users
+type manualCheck struct {
+	Description string
+	If          c.Condition // If the condition is met, a checkbox will be displayed on bot comment
+	Teams       []string    // Members of these teams can check the checkbox to make the check pass
+}
+
+// This function returns the configuration of the bot consisting of automatic and manual checks
+// in which the GitHub client is injected
+func config(gh *client.GitHub) ([]automaticCheck, []manualCheck) {
```

Comment:
Yes, you're right, but that was the idea of the bot as discussed during the call where we defined the specs: to avoid trying to create something generic, focus only on `gnolang/gno` and to hardcode the configuration in a Go file rather than in a JSON or YAML file. 

However, indeed, if someone wants to reuse the bot, in its current state, they will need to modify the `config.go` file and recompile the bot for it to work. 

We could eventually consider making this bot importable and customizable.

Reply:
I will put everything package in an internal folder to make things clear.